vars/dockerExecute.groovy

```diff
@@ -169,8 +178,14 @@ void call(Map parameters = [:], body) {
                     paramMap.containerWorkspaces[config.dockerImage] = config.dockerWorkspace
                     paramMap.containerWorkspaces[config.sidecarImage] = ''
 
+                    echo config.toString()
```

Comment:
let's also introduce `verbose` option as done in many other steps already and only echo if verbose is set to true

Reply:
Sry. Just a debug left over. Removed